formats/arrow-to-tabular-plugin/src/polus/plugins/formats/arrow_to_tabular/arrow_to_tabular.py

```diff
@@ -0,0 +1,47 @@
+"""Arrow to Tabular."""
+import logging
+import pathlib
+
+import pyarrow.feather as pf
+import vaex
+
+logger = logging.getLogger(__name__)
+
+
+def arrow_tabular(file: pathlib.Path, file_format: str, out_dir: pathlib.Path) -> None:
+    """Convert Arrow file into tabular file using pyarrow.
+
+    Args:
+        file (Path): Path to input file.
+        file_format (str): Filepattern of desired tabular output file.
+        out_dir (Path): Path to output directory.
+    Returns:
+        Tabular File
```

Comment:
This function doesn't return anything. Instead, it would be a good idea to include some additional description in the main paragraph of the docstring. I think if you make the a file format enum, that should make things easier to explain.